arch/cpu/cc26xx-cc13xx/rf-core/ble-hal/rf-ble-cmd.c

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
Is this module entirely cc26xx-specific? Nothing that would be reused in ports to other platforms?

Reply:
Yes, this module and its commands are specific to the TI CC26xx radio.
These commands cannot be reused on other platforms with BLE support